reframe/core/meta.py

```diff
@@ -486,14 +474,16 @@ def __call__(cls, *args, **kwargs):
         # respective points in the parameter and fixture spaces.
         variant_num = kwargs.pop('variant_num', None)
         param_index, fixt_index = cls._map_variant_num(variant_num)
+        fixt_name = kwargs.pop('fixt_name', None)
+        fixt_data = kwargs.pop('fixt_data', None)
 
         # Intercept variables to be set before initialization
-        variables = kwargs.pop('variables', {})
-        if not isinstance(variables, collections.abc.Mapping):
-            raise TypeError("'variables' argument must be a mapping")
+        fixt_vars = kwargs.pop('fixt_vars', {})
+        if not isinstance(fixt_vars, collections.abc.Mapping):
+            raise TypeError("'fixt_vars' argument must be a mapping")
 
         # Intercept is_fixture argument to flag an instance as a fixture
-        is_fixture = kwargs.pop('is_fixture', False)
+        # is_fixture = kwargs.pop('is_fixture', False)
```

Comment:
I am not sure what are the implications but in the end we should remove the comment if we don't want to use it.

Reply:
It's just forgotten 😅

Reply:
Whether a test is a fixture or not is now determined differently and this is not needed. We store the whole `FixtureData` object in the test.